pynucastro/eos/fermi_integrals.py

```diff
@@ -0,0 +1,247 @@
+# In this particular module, we write the routines from Aparicio (1998) and Gong (2001),
+# described in https://iopscience.iop.org/article/10.1086/313121. and
+# https://www.sciencedirect.com/science/article/abs/pii/S001046550100145X?via%3Dihub.
+# The method consist on breaking down the Fermi-integrals (eta, theta) integration domain
+# into four subintervals and apply Gauss-Legendre or the Gauss laguerre quadrature methods
+# with 20 points over each domain to guarantee accuracy beyond double-precision.
+
+# In the first subinterval, a x=z^2 change of variable is applied to overcome the kernel
+# singularity near the origin.
+
+import numpy as np
+from scipy.special import roots_laguerre, roots_legendre
+
+
+class FermiIntegrals:
```

Comment:
since the class takes `k` in the init, I think it is more appropriate to call the class `FermiIntegral` (not plural), since we are defining exactly what integral we are doing on construction.

Reply:
this also needs docstrings in the numpydoc format